os/board/rtl8730e/src/rtl8730e_ndp120_iwrl6432.c

```diff
@@ -209,12 +230,12 @@ static void rtl8730e_iwrl6432_enable_irq(bool enable)
 	}
 }
 
-static void rtl8730e_iwrl6432_irq_attach(ndp120_handler_t handler, FAR char *arg)
+static void rtl8730e_iwrl6432_irq_attach(iwrl6432_handler_t handler, FAR char *arg)
 {
 	g_iwrl6432info.handler = handler;
 	gpio_irq_init(&g_iwrl6432info.data_ready, GPIO_IWRL6432_PIN_BUSY, rtl8730e_iwrl6432_irq_handler, arg);
-	gpio_irq_set(&g_iwrl6432info.data_ready, IRQ_LOW, 1);
-	gpio_irq_enable(&g_iwrl6432info.data_ready);
+	gpio_irq_set(&g_iwrl6432info.data_ready, IRQ_FALL_RISE, 1);
```

Comment:
I really worry about this, we couldn't use IRQ_FALL_RISE because so many times it skipped when we use bixby in product..... iwrl6432 is one of external module, so H/W team will request aliveness logic, and QA will test this with much of pulse noise.